src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -769,6 +773,29 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 					// to get the behaviour people expect.
 					AiNodeSetArray( node, "trace_sets", AiArray( 1, 1, AI_TYPE_STRING, "__none__" ) );
 				}
+
+				if( m_linkedLights )
+				{
+					const std::vector<std::string> &lightNamesWritable = m_linkedLights->readable();
```

Comment:
Ah, yeah - I used `->writable()` before and forgot to fix the variable name when I fixed the method call.